scipy/stats/tests/test_distributions.py

```diff
@@ -1377,6 +1377,34 @@ def test_sf_isf(self):
         xx = stats.gumbel_l.isf(y)
         assert_allclose(x, xx)
 
+        
+class TestGammaGompertz(object):
+    def test_pdf(self):
+        vals = stats.gamma_gompertz.pdf(np.log(np.arange(1., 6., 1.)), 1., 2.)
+        expected = [0.5, 4./9., 0.375, 8./25., 5./18.]
```

Comment:
Is `0.375` the same as `3.0/8`?  If they all are rational numbers, present them as rational, rather than a mix of decimal and rational.  The same comment applies below as well.

Reply:
Thanks for your suggestion. Accordingly,
```
        expected = [0.5, 4./9., 0.375, 8./25., 5./18.]
```
needs to be changed to
```
        expected = [1./2., 4./9., 3./8., 8./25., 5./18.]
```